bikeshed/test.py

```diff
@@ -12,18 +12,35 @@
 from .messages import *
 
 
+TEST_DIR = os.path.abspath(os.path.join(config.scriptPath(), "..", "tests"))
```

Comment:
Just `os.path.abspath(config.scriptPath() + "/../tests")` would probably have worked as well because of abspath, but I went for eliminating path separators.